assets/js/base/components/drawer/style.scss

```diff
@@ -18,7 +21,7 @@ $drawer-width-mobile: 100vw;
 	}
 
 	to {
-		transform: translateX(-$drawer-width);
+		transform: translateX(var(--neg-drawer-width));
```

Comment:
Same change here:
```suggestion
		transform: translateX(max(-100%, var(--neg-drawer-width)));
```

Then, I think we can completely get rid of `$drawer-width-mobile`? :thinking:

Reply:
Yes, I think so. It should be possible to remove those.